### PR TITLE
Add new job for GitHub Actions to label pull request automatically

### PR DESCRIPTION
**Pull request by andersonjeccel**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 

## Description

This PR adds a labeler job to:
- Automatically assign labels depending of the modified bundle
- Add needs-triage to every new PR
- Add 'bug' label for PR that contain bug or fix in their name

---
### 📋 Steps to test this PR:

I'm not sure here

### CODE REVIEW

**Comment by andersonjeccel** [comment]
@escopecz What do you think?

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  - "app/bundles/NotificationBundle/**"
+  
+plugin:
+  - "app/bundles/PluginBundle/**"
+  
+points-scoring:
+  - "app/bundles/PointBundle/**"
+  
+queue:
+  - "app/bundles/QueueBundle/**"
```
Comment (escopecz) [inline .github/labeler.yml]: The queue bundle will be removed in M6 and replaced by Symfony's way how to do queues. So perhaps we don't need it here as it won't get any adjustments anymore.

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  
+reports:
+  - "app/bundles/ReportBundle/**"
+  
+roles:
+  - "app/bundles/UserBundle/**"
```
Comment (escopecz) [inline .github/labeler.yml:62]: A change in the UserBundle doesn't necessarily mean it is a roles change. Roles are just a part of the user bundle/

**Review by escopecz — COMMENTED** [review]
Is there a way how to test this before it's merged?

File: .github/labeler.yml (hunk 2 above)
Comment (andersonjeccel) [reply .github/labeler.yml:62]: I think we use the Roles label both for roles and changes related to users (as we don’t have a users label)

Is that what you’re talking about?

If not, how could we improve?

**Comment by andersonjeccel** [comment]
@escopecz the labeler appears on the tests list below the composer lock check for this PR
It’s passing

I’m not sure if there’s another way

**Review by escopecz — DISMISSED** [review]
Let's give it a try 👍

File: .github/labeler.yml (hunk 1 above)
Comment (andersonjeccel) [reply .github/labeler.yml]: Removed

**Review by RCheesley — APPROVED** [review]
Thanks @andersonjeccel - good to go, let's see how it shakes out.

**Comment by escopecz** [comment]
@andersonjeccel this is failing CI on open PRs with
```
The configuration file (path: .github/labeler.yml) is found locally, reading from the file
```
Example: https://github.com/mautic/mautic/pull/13990/checks

Do you want to quickly fix that or do you want me to revert?

**Comment by andersonjeccel** [comment]
@escopecz Let me try a fix

**Comment by escopecz** [comment]
Reverted in https://github.com/mautic/mautic/pull/13993

**Pull request merged**